### PR TITLE
Issue/#109 setup iexcloud

### PR DESCRIPTION
**Pull request by razine-bensari**
# Related Issue
- #109 

# Proposed changes
- Setup IEXCloud for stock user story

# Checklist (if applicable)
- [ ] Tests not yet. Will be updated with PR commit

# Reviewer(s)
 - @Alessandro100 @razine-bensari  @abdulmansour @DPigeon

### CODE REVIEW

File: backend/src/main/java/com/capstone/moneytree/controller/StockController.java [hunk 1]
```diff
@@ -1,23 +1,83 @@
...
-      this.stockService = stockService;
+   /**
+    * Obtain json object containing company quote, info, news, keystats and logo
+    *
+    * @param symbol
+    * @return batchStocks
+    */
+   @GetMapping("/batch/{symbol}")
+   public ResponseEntity<BatchStocks> getBatchStocksBySymbol(@Valid @PathVariable(name = "symbol") String symbol) {
+      BatchStocks batchStocks = stockMarketDataService.getBatchStocksBySymbol(symbol);
```
Comment (razine-bensari) [inline backend/src/main/java/com/capstone/moneytree/controller/StockController.java:37]: Is there a way to convert this BatchStock from IEXCloud library to the Stock class that we actually persist in NEO4J.

So front end can simply receive our Stock object. WDYT?

Comment (razine-bensari) [reply backend/src/main/java/com/capstone/moneytree/controller/StockController.java:37]: Actually, now that I think about it. There is no need. Stock class is the stock bought by the user. BatchStock is what the front end needs with all info about stocks

**Review by DPigeon — APPROVED** [review]
Nice work! Approved.

**Review by Alessandro100 — APPROVED** [review]
Looks like good work, the one thing to investigate is how sonarcloud works with our coverage as it says 0 but we do have tests. This can be done in a later commit

**Pull request merged**